src/index.js

```diff
@@ -1,12 +1,23 @@
+import AddScore from './modules/addScore.js';
+import Game from './modules/game.js';
 import List from './modules/list.js';
 import RecentScores from './modules/recentScores.js';
 import Score from './modules/score.js';
+import Store from './modules/store.js';
 import './style.scss';
 
-const list = new List();
+export const URL_BASE = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
 
-[100, 20, 50, 78, 125, 77, 42].forEach((score) => {
-  list.add(new Score('Name', score));
-});
+(async () => {
+  const game = localStorage.getItem('game') || (await (await Game.init('Pokemon')).json()).result.slice(14, 34);
+  localStorage.setItem('game', game);
 
-document.querySelector('.content-1').innerHTML = new RecentScores(list).render();
\ No newline at end of file
+  document.querySelector('.submit').onclick = (e) => {
+    e.preventDefault();
+    AddScore.add(new Score(), game);
+  };
+  document.querySelector('.refresh').onclick = async () => {
+    window.location.reload();
+  };
```

Comment:
[Optional] It would be better if the webpage was not refreshed whenever the refresh button is clicked. This will improve your user experience. You can add a loading state instead😎